example_8/test/test_urdf_xacro.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022 FZI Forschungszentrum Informatik
```

Comment:
Please remove license,  just add instition and author name, and refer to the license in the repository's root.